ext/opcache/jit/zend_jit_arm64.dasc

```diff
@@ -9220,6 +9220,13 @@ static int zend_jit_do_fcall(dasm_State **Dst, const zend_op *opline, const zend
 					|	MEM_LOAD_64_ZTS ldr, REG2, compiler_globals, map_ptr_base, TMP1
 					|	ADD_SUB_64_WITH_CONST add, REG2, REG2, (uintptr_t)ZEND_MAP_PTR(func->op_array.run_time_cache), TMP1
 					|	ldr REG2, [REG2]
+				} else if ((func && (func->op_array.fn_flags & ZEND_ACC_CLOSURE)) ||
+						(JIT_G(current_frame) &&
+						 JIT_G(current_frame)->call &&
+						 TRACE_FRAME_IS_CLOSURE_CALL(JIT_G(current_frame)->call))) {
+					/* Closures always use direct pointers */
+					|	ldr REG2, [REG0, #offsetof(zend_op_array, run_time_cache__ptr)]
+					|	ldr REG2, [REG2]
```

Comment:
Is this second `ldr` right? Doesn't the previous line already load the final run time cache pointer?